src/index.ts

```diff
@@ -1281,8 +1281,8 @@ export class Stream<T> implements InternalListener<T> {
    * @factory true
    * @return {Stream}
    */
-  static never(): Stream<any> {
-    return new Stream<any>({ _start: noop, _stop: noop });
+  static never<T = any>(): Stream<any> {
```

Comment:
Shouldn't this function return `Stream<T>` now?

Reply:
Yes, my bad.